composer.json

```diff
@@ -58,7 +58,7 @@
         "symfony/http-foundation": "~2.3",
         "symfony/form": "~2.3",
         "symfony/validator": "~2.6",
-        "ajbdev/authorizenet-php-api": "1.1.*",
+        "authorizenet/authorizenet": "1.8.*",
```

Comment:
You have to update this line https://github.com/Payum/Payum/pull/417/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L93 too, and the composer.json of Payum/AuthorizeNet/Aim component